modules/javafx.controls/src/main/java/com/sun/javafx/scene/control/ContextMenuContent.java

```diff
@@ -810,6 +819,14 @@ Menu getOpenSubMenu() {
         return openSubmenu;
     }
 
+    boolean isUpArrowVisible() {
+        return upArrow.isVisible();
+    }
+
+    boolean isDownArrowVisible() {
+        return upArrow.isVisible();
```

Comment:
Just to double-check, shouldn't this be `downArrow.isVisible()`?

Reply:
Yes it should be `downArrow.isVisible()`. I'll update the code.

Reply:
While this is a simple change, I thought of adding more checks on the visibility of both up and down arrow in the test to avoid these types of error in code. I'm seeing a strange issue that, I'm getting failures when more than 4 `Assert.assertTrue/assertFalse` statements are added. I'll investigate more on this and then update the code.

Reply:
Updated code and added more checks on visibility of up and down arrow in system test.